lib/Common/Common/NumberUtilities_strtod.cpp

```diff
@@ -1883,7 +1883,6 @@ static BOOL FDblToRgbFast(double dbl, _Out_writes_to_(kcbMaxRgb, (*ppbLim - prgb
 
 static BOOL FormatDigits(_In_reads_(pbLim - pbSrc) byte *pbSrc, byte *pbLim, int wExp10, _Out_writes_(cchDst) OLECHAR *pchDst, int cchDst)
 {
-    AssertArrMem(pbSrc, pbLim - pbSrc);
     AnalysisAssert(pbLim > pbSrc);
 
```

Comment:
Shouldn't static analysis figure this out for the code after the block in lines 1888-1892?

Reply:
I believe that static analysis got confused before and this was added.